bridges/otelzap/core_test.go

```diff
@@ -51,8 +51,22 @@ func TestCore(t *testing.T) {
 
 	rec.Reset()
 
-	// TODO: Add WriteContext test case.
-	// TODO: Add WithContext test case.
+	t.Run("Write Context", func(t *testing.T) {
+		ctx := context.Background()
+		ctx = context.WithValue(ctx, testEntry, true)
+		logger.Info(testMessage, zap.Reflect("ctx", ctx))
```

Comment:
It should work with `zap.Any` as well. Let use it in one test case to have it clear.

```suggestion
		logger.Info(testMessage, zap.Any("ctx", ctx))
```